src/modules/tkl.c

```diff
@@ -3706,7 +3706,7 @@ void tkl_sync_send_entry(int add, Client *sender, Client *to, TKL *tkl)
 	} else
 	{
 		unreal_log(ULOG_FATAL, "tkl", "BUG_TKL_SYNC_SEND_ENTRY", NULL,
-			   "[BUG] tkl_sync_send_entry() called for '%s' but unknown type: $tkl.type_string ($tkl_type_int)",
+			   "[BUG] tkl_sync_send_entry() called, but unknown type: $tkl.type_string ($tkl_type_int)",
```

Comment:
UnrealIRCd 5.2.x matches the message I've changed it to. I was a bit unclear to what the %s could expand out to (given $tkl.type_string and $tkl_type_int are already present)